File: Day1/part1.py
#!/usr/bin/env python3

with open('input', 'r') as f:
    max  = 0
    deer = 0
    for l in f:
        l = l.rstrip()
        if len(l):
            deer = deer + int(l);
        else:
            if max < deer:
                max = deer
            deer = 0
    print(max)
    

File: Day1/part1.pl
#!/usr/bin/env perl

use 5.030;
use strict;
use warnings;

open my $fh, '<', 'input' or die "Failed to open input file: $!";
my $max  = 0;
my $deer = 0;
while (defined(my $l = readline $fh)) {
    chomp $l;
    if (length $l) {
        $deer += $l;
    }
    else {
        $max < $deer and $max = $deer;
        $deer = 0;
    }
}
say $max;